client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/SutInfoDto.java

```diff
@@ -12,6 +13,7 @@
      * on how to interact with it
      */
     public RestProblemDto restProblem;
+    public GraphqlProblemDto graphqlProblem;
```

Comment:
add an empty line (for readability), and then some JavaDocs like in `restProblem`